protostar/testing/cheatcodes/declare_cheatcode.py

```diff
@@ -59,8 +59,16 @@ def declare(
 
         return DeclaredContract(class_hash)
 
-    async def _declare_contract(self, contract_path: Path):
-        contract_class = self._starknet_compiler.compile_contract(contract_path)
+    async def _declare_contract(self, contract: str):
+        contract_identifier = (
+            Path(contract).resolve() if contract.endswith(".cairo") else contract
+        )
```

Comment:
couldn't this logic be shared with migrator's cheatcodes? for example by putting it inside `ProjectCompiler` class?

Reply:
I will extract recognition of `.cairo` ending strings as cairo source files to `ProjectCompiler`. The second is out of scope of this PR and can be discussed/considered separately if we really want it, since it'd be modification of migrator's behavior.

Reply:
^done